packages/bloc/lib/src/bloc_observer.dart

```diff
@@ -3,6 +3,9 @@ import 'package:meta/meta.dart';
 
 /// An interface for observing the behavior of [Bloc] instances.
 abstract class BlocObserver {
+  /// An interface for observing the behavior of [Bloc] instances.
```

Comment:
Can we create a template and use a macro here instead of copy/pasting the same documentation?